firefox-ios/Client/Frontend/Home/Homepage Rebuild/HomepageViewController.swift

```diff
@@ -552,7 +552,10 @@ final class HomepageViewController: UIViewController,
             }
 
             if let storyCell = storyCell as? StoryCell {
-                storyCell.configure(story: story, theme: currentTheme)
+                let position = indexPath.item + 1
+                let currentSection = dataSource?.snapshot().sectionIdentifiers[indexPath.section] ?? .pocket(.clear)
+                let totalCount = dataSource?.snapshot().numberOfItems(inSection: currentSection) ?? 0
```

Comment:
Given that we're unwrapping in the provided function, we don't really need a default value here. Also, we don't want to risk some weird case where it start saying "3 of 0", so an optional is totally fine here.